excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/endpoint/exception/RestResponseEntityExceptionHandler.java

```diff
@@ -46,6 +45,12 @@ public RestResponseEntityExceptionHandler(SpreadsheetValidationErrorMessageCreat
         return handleExceptionInternal(ex, flattenExceptionMessages(ex), new HttpHeaders(), HttpStatus.BAD_REQUEST, request);
     }
 
+    @ExceptionHandler(value = MissingUserRolesException.class)
+    ResponseEntity<Object> handleUserRolesMissing(MissingUserRolesException ex, WebRequest request) {
+        log.error("Missing UserRoles '{}'", ex.getMessage(), ex);
```

Comment:
let's log as warn and perhaps without stacktrace?

Reply:
Changed it... not logging stacktrace.. just logging message